src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Rultor-DynamoKey: ${dynamo.key}
 Rultor-DynamoSecret: ${dynamo.secret}
 Rultor-GithubId: ${github.id}
 Rultor-GithubSecret: ${github.secret}
+Rultor-GithubThreshold: ${github.threshold}
```

Comment:
@yegor256 This threshold isn't the max requests quota, but rather the number of remaining requests at or below which the CarefulWire will sleep until the quota gets replenished